### PR TITLE
Pangea - COVID19 in Italy

### PR DESCRIPTION
**Pull request by pangeaCovid19**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/pangea-Covid19.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

File: contest/contributions/pangea-Covid19.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: Pangea - COVID19 in Italy
+author: Pangea
```
Comment (riccardoporreca) [inline contest/contributions/pangea-Covid19.yml]: We expect `author` to be a person's name, put you can of course keep Pangea
```suggestion
author: Andrea Romualdi, Pangea
```

File: contest/contributions/pangea-Covid19.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+abstract: |
+  PangeaCovid19 is an interactive tool that aims to bring insights on the spread of Covid19 in Italy. It is divided in 5 sections  in Previsioni we show short term forecasts for the total counts of infected people and deaths obtained with 3 different statistical models;  in Terapia Intensiva we show the occupation of intensive care units by covid19 patients on a regional basis;  in Per Regione/Per Provincia we show the trend several variables (new cases, deaths, swabs, number of tested people) on a regional basis allowing a comparison between regions;  in Matematica della diffusione we explain why we use different forecasting models;  in Diario you can find some articles and daily diaries to monitor the situation.  Data are provided by Protezione Civile, ISTAT and Ministero della Salute.
+menu_entry: PangeaCovid19
+repository: https://github.com/user/repo
```
Comment (riccardoporreca) [inline contest/contributions/pangea-Covid19.yml]: Being an open source contest, a link to a public source code is mandatory. Please provide one (doesn't have to be a GitHub repo).

**Review by riccardoporreca — CHANGES_REQUESTED** [review]
@pangeaCovid19, thank you for your submission.

It is important that contributions are accessible to an international audience, therefore we can only accept work that is approachable and can be understood by English-speaking users.

Therefore we recommend to update the app by providing some **minimal English localization** of your work.
See also below for more specific comments to be addressed in the provided YAML fields.

If I can suggest something concrete that does not require a major localization effort (with <1 day left), you could:
- add clickable elements (e.g. a :gb: flag or a text "English explanation") to **each plot** (e.g. at the end of the plot title like 'Andamenti globali in Italia con previsione a 3 giorni').
- the clickable element opens a **modal window** where you provide an English explanation of what the specific plot is about (a translation of the plot title and the details below the plot would probably do the job). If you are new to modals, package [bsblus](https://ijlyttle.github.io/bsplus/articles/modal.html) makes it very easy and natural.

This would be the bare minimum for your contribution to be accepted.

Looking forward to include your work in the contest after your updates.

File: contest/contributions/pangea-Covid19.yml (hunk 2 above)
Comment (pangeaCovid19) [reply contest/contributions/pangea-Covid19.yml]: Thank you Riccardo, 

I have a pair of questions:
- Is  There a specific time for the deadline.
- How many authors can I enter most?

Comment (riccardoporreca) [reply contest/contributions/pangea-Covid19.yml]: @pangeaCovid19
- The deadline is May 22nd 23:59 CEST.
- Nr of authors: 4, use 4+ et al. if you want to explicit about this being authored by more.

**Comment by riccardoporreca** [comment]
Great job @pangeaCovid19 :clap:!

There are currently issues with GitHub preventing the integration of contributions (https://www.githubstatus.com/).
We will finalize submissions once such issues are solved.

**Review by riccardoporreca — APPROVED** [review]

File: contest/contributions/pangea-Covid19.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+title: Pangea - COVID19 in Italy
+author: Nicola Farina, Fabio Priuli, Andrea Romualdi, Martina Tuccinardi et. al - Pangea
+abstract: |
+  PangeaCovid19 is an interactive tool that aims to bring insights on the spread of Covid19 in Italy. It is divided in 4 sections  in Previsioni we show short term forecasts for the total counts of infected people and deaths obtained with 3 different statistical models;  in Terapia Intensiva we show the occupation of intensive care units by covid19 patients on a regional basis;  in Per Regione/Per Provincia we show the trend several variables (new cases, deaths, swabs, number of tested people) on a regional basis allowing a comparison between regions;  in Matematica della diffusione we explain why we use different forecasting models.  Data are provided by Protezione Civile, ISTAT and Ministero della Salute.
+menu_entry: PangeaCovid19
+repository: https://github.com/pangeaCovid19/Covid19eRUM.git
+type: shiny
+content: https://www.pangeadds.eu/demos/CoVid19_eng/
```
Comment (riccardoporreca) [inline contest/contributions/pangea-Covid19.yml:8]:
```suggestion
content: https://www.pangeadds.eu/demos/CoVid19_eng/
date: 2020-05-22-23:59
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
@pangeaCovid19, the major GitHub incident was solved and now contributions are fully integrated.

Your contribution is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/pangea-Covid19.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**